### PR TITLE
fix #100221: changed checkbox to radio buttons

### PR DESCRIPTION
**Pull request by salilkhandelwal**
Made page settings "Landscape" check box into proper "portrait/landscape" radio buttons

### CODE REVIEW

File: mscore/pagesettings.ui [hunk 1]
```diff
@@ -151,8 +147,19 @@
           </property>
          </widget>
         </item>
-        <item row="0" column="0" colspan="2">
-         <widget class="QComboBox" name="pageGroup"/>
+        <item row="3" column="0">
+         <widget class="QRadioButton" name="portraitButton">
+          <property name="text">
+           <string>portrait</string>
```
Comment (Jojo-Schmitz) [inline mscore/pagesettings.ui]: "Portrait" pease, capital P

File: mscore/pagesettings.ui [hunk 2]
```diff
@@ -151,8 +147,19 @@
...
+          </property>
+         </widget>
+        </item>
+        <item row="4" column="0">
+         <widget class="QRadioButton" name="landscapeButton">
+          <property name="text">
+           <string>landscape</string>
```
Comment (Jojo-Schmitz) [inline mscore/pagesettings.ui]: "Landscape" please, capital L

File: mscore/pagesettings.ui (hunk 1 above)
Comment (salilkhandelwal) [inline mscore/pagesettings.ui]: I wanted to ask whether you want me to change "portraitButton" to "PortraitButton" as well ?
Currently I am thinking of changing "<string>portrait</string>"to <string>Portrait</string>.

Comment (Jojo-Schmitz) [inline mscore/pagesettings.ui]: Yes, only the user visible (and translatable) strings

**Comment by IsaacWeiss** [comment]
@lasconic, any reason this was skipped?

**Comment by IsaacWeiss** [comment]
Is there any reason this hasn't been merged yet?

**Pull request merged**